shared/build.gradle.kts

```diff
@@ -55,6 +57,13 @@ kotlin {
         val commonTest by getting {
             dependencies {
                 implementation(kotlin("test"))
+                with(Dependencies.Test) {
+                    implementation(COROUTINES)
+                    implementation(KOTEST_FRAMEWORK)
+                    implementation(KOTEST_ASSERTIONS)
+                    implementation(KOTEST_PROPERTY)
+                    implementation(MOCKATIVE)
```

Comment:
You might need to check this tool: https://github.com/kosi-libs/MocKMP#with-the-official-plugin

Bliss is using that, compare to my KMM-IC, it's better, it has the `fake` annotation that's can support faking models as well.

Reply:
@suho Coming from [mockk](https://mockk.io/) on Android, I prefer the `mockative` style rather than `MocKMP`. The setup logic of MocKMP does not look fine to me with `injectMocks` and `withMocks` utils 🤷‍♂️ .

With fake, I still feel good with MockUtil class where I can clarify my models in detail 🤔

Reply:
The best is mockk can support KMM https://github.com/mockk/mockk/issues/58 =))